exercises/practice/grains/.docs/instructions.md

```diff
@@ -13,14 +13,15 @@ There are 64 squares on a chessboard (where square 1 has one grain, square 2 has
 
 Write code that shows:
 - how many grains were on a given square, and
-- the total number of grains on the chessboard
+- the total number of grains on the chessboard,
+- and throws a Value Error for numbers over 64.
```

Comment:
Many of the Exercism exercises hides the expected implementation details inside the unit tests. If we want to spell out the expectations in details, that should be an Exercism-wide conversation first.